config/policy-controller/rbac/leader_election_role.yaml

```diff
@@ -0,0 +1,45 @@
+# permissions to do leader election.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  labels:
+    app.kubernetes.io/name: role
+    app.kubernetes.io/instance: policy-controller-leader-election-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: policy-controller
+    app.kubernetes.io/part-of: kuadrant
+    app.kubernetes.io/managed-by: kustomize
+  name: policy-controller-leader-election-role
+  namespace: system
+rules:
```

Comment:
We should probably get rid of all these labels.

Reply:
Reason being, they look odd here https://github.com/Kuadrant/kuadrant-operator/pull/293/files#diff-d40f82c17f85444a248492d3d46940851c2bf53af0e6c00dfd3e8e2782e0f936R8. You'd need to patch then there, but not sure they offer much anyway. Think they were added by kubebuilder during generation.

Reply:
got rid of them in the policy controller